tensorflow/lite/kernels/fully_connected.cc

```diff
@@ -393,11 +429,7 @@ TfLiteStatus EvalShuffledQuantized(TfLiteContext* context, TfLiteNode* node,
                                    TfLiteTensor* shuffled_input_workspace) {
   gemmlowp::GemmContext* gemm_context = gemm_support::GetFromContext(context);
 
-  // TODO(b/110697972) decide more consistently if / how / where we want
```

Comment:
Please leave this TODO, since its referring to a more general policy question, of where we should be checking types.